sampler/stories/default/Heading.js

```diff
@@ -0,0 +1,32 @@
+import Heading, {HeadingBase} from '@enact/agate/Heading';
+
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import {boolean, select, text} from '../../src/enact-knobs';
+import {mergeComponentMetadata} from '../../src/utils';
+
+Heading.displayName = 'Heading';
+const Config = mergeComponentMetadata('Heading', Heading, HeadingBase);
+
+const prop = {
+	colors: ['', '#E6444B', '#FDC902', '#986AAD', '#4E75E1', '#30CC83', '#44C8D5', '#47439B', '#2D32A6', '#4E75E1'],
+	sizes: ['', 'small', 'medium', 'large', 'title', 'none']
```

Comment:
I think these are the values for `spacing` prop and `sizes` should be the values for `ui/Heading`:


`sizes: ['', 'tiny', 'small', 'medium', 'large', 'subtitle', 'title'],`
`spacing: ['auto', 'title', 'large', 'medium', 'small', 'none']`

Reply:
Agate Heading doesn't currently support the 'subtitle' size. (Nothing like that in the designs yet.)